docs/cli/create.md

```diff
@@ -0,0 +1,88 @@
+# Create
+This command used for creating DID.
+
+## Usage
+```
+create-did [flags]
+```
+
+## Flags
+* `domain` _[string]_ - URL to the TrustBloc consortium's domain.
+* `sidetree-url` _[array|string]_ - Array of one or more Sidetree URLs.
+* `sidetree-write-token` _[string]_ - The Sidetree write token.
+* `tls-cacerts ` _[array|string]_ - Array of one or more CA cert paths.
+* `tls-systemcertpool ` _[boolean]_ - Flag whether to use system certificate pool.
+* `publickey-file` _[string]_ - The file contains the DID public keys.
+* `service-file` _[string]_ - The file contains the DID services.
+* `recoverykey` _[string]_ - The public key PEM used for recovery of the document.
+* `recoverykey-file` _[string]_ - The file that contains the public key PEM used for recovery of the document.
+* `updatekey` _[string]_ - The public key PEM used for validating the signature of the next update of the document.
+* `updatekey-file` _[string]_ - The file that contains the public key PEM used for validating the signature of the next update of the document.
+
+## Example
+
+### create cmd
+```
+create-did --domain testnet.trustbloc.local --publickey-file ./publickeys.json --service-file ./services.json 
+--recoverykey-file ./keys/recover/public.pem --updatekey-file ./keys/update/public.pem
+```
+
+### publickeys.json
+```
+[
+ {
+  "type": "Ed25519VerificationKey2018",
+  "purposes": ["authentication"],
+  "jwkPath": "./key1_jwk.json"
+ },
+ {
+  "type": "JwsVerificationKey2020",
+  "purposes": ["capabilityInvocation"],
+  "jwkPath": "./key2_jwk.json"
+ }
+]
+```
+
+### key1_jwk.json
+```
+{
+  "kty":"OKP",
+  "kid": "key1",
```

Comment:
Is this kid used to identify public key id in the document?

Reply:
Yes